src/dso_api/dynamic_api/routers.py

```diff
@@ -135,7 +142,13 @@ def initialize(self):
 
     def get_urls(self):
         """Add extra URLs beside the registered viewsets."""
-        return super().get_urls() + self._openapi_urls + self._index_urls
+        return (
+            super().get_urls()
+            + self._openapi_urls
+            + self._index_urls
+            + self._mvt_urls
+            + self._wfs_urls
+        )
```

Comment:
This is not performance critical code but instead of creating 5 separate lists, only to throw away 4 of them feels not very Pythonic to me. I'd rather see:

```python
from itertools import chain

return list(chain(
    super().get_urls(),
    self._openapi_urls,
    self._index_urls,
    self._mvt_urls,
    self._wfs_urls))
```